.github/workflows/plugin-release.yml

```
@@ -0,0 +1,145 @@
+      - 'PKHeX.WinForms/MainWindow/Plugins/**'
+      - 'PKHeX.Core/Resources/legality/**'
+    branches:
+      - main
+      - master
+
+jobs:
+  build-and-release:
+    runs-on: windows-latest
+    
+    steps:
+    - name: Checkout code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)